contracts/Governance/TransactionRelay.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract TransactionRelay is Ownable {
+    event CreateTransaction(
+        bytes32 indexed txHash,
+        address indexed creator,
+        address indexed targetContractAddress,
+        uint256 value,
+        string signature,
+        bytes data
+    );
+    event DeleteTransaction(
+        bytes32 indexed txHash,
+        address indexed creator,
+        address indexed targetContractAddress,
+        uint256 value,
+        string signature,
+        bytes data
+    );
+    event ExecuteTransaction(
+        bytes32 indexed txHash,
+        address indexed executioner,
+        address creator,
+        address indexed targetContractAddress,
+        uint256 value,
+        string signature,
+        bytes data,
+        bytes returnData
+    );
+
+    mapping(bytes32 => StoredTransaction) public storedTransactions;
+
+    struct StoredTransaction {
+        address creator;
+        address targetContractAddress;
+        uint256 value;
+        string signature;
+        bytes data;
+    }
+
+    constructor() public {}
```

Comment:
is it required to define a constructor event if it's empty?

Reply:
nope